policies/access.md

```diff
@@ -0,0 +1,35 @@
+# GitHub Access Management Policy
+
+## Teams, not Individuals
+
+Access to repositories will be granted only to GitHub teams, not to individuals. A repo may have any number of teams added to it, with the appropriate access levels.
+
+## Team Structure
+
+Teams will be nested, using parent/child relationships, as needed. A child team will implicitly have any privileges its parent team does, so, the most deeply nested team should be the one with the most privileges.
+
+### Top-level teams
+
+Note: this list is intentionally not exhaustive.
+
+- [TAC](https://github.com/orgs/ossf/teams/tac)
+    This team is for current TAC members. Individuals should be added or removed from this team to reflect current membership.
+- [Staff](https://github.com/orgs/ossf/teams/staff)
+    This team is for OpenSSF staff members. Individuals on the top-level team are there as a catch-all, but may be moved to subteams as the need arises.
+  - [PMs](https://github.com/orgs/ossf/teams/pms)
+      This teams is for PMs, who often need Maintain or Admin access on repos.
+  - [Marketing](https://github.com/orgs/ossf/teams/marketing)
+- [Working Groups](https://github.com/orgs/ossf/teams/working-groups)
+    This is the parent team for Working Groups. Every WG should have a subteam contained within this one. All WG subteams must start with `wg-` for consistency.
+- [SIGs](https://github.com/orgs/ossf/teams/sigs)
+    This is the parent team for SIGs. Every SIG should have a subteam contained within this one. All SIG subteams must start with `sig-` for consistency.
+- [Projects](https://github.com/orgs/ossf/teams/projects)
+    This is the parent team for projects. Every project (eg scorecard, AO) should have a subteam contained within this one.
+    Teams for individual repositories go under here, which start with `repo-`, but team names may otherwise be unconstrained.
```

Comment:
Following PLP(below) projects will need teams for each access level (eg scorecard-read, scorecard-traige, scorecard-write, scorecard-maintain)

Reply:
In the extreme yes, but we'd only create these as needed, since most repos don't need this granularity atm.

Reply:
I don't think this is just an extreme case. Take a look at the BP-WG group:
https://github.com/orgs/ossf/teams/wg-best-practices/members
https://github.com/orgs/ossf/teams/wg-best-practices/repositories

All those people who had some kind of access to any of the 6 repos now have write/maintain access to all of them. In an effort to reduce the manual configuration of groups. repos and access-levels will be consolidated and people will be getting access they don't need and/or shouldn't have.

Reply:
Sure, that's a fair point. However, when I created that specific team i was told that all members should have Write. There's a tradeoff here balancing convenience and "least privilege", and since Write is a pretty non-dangerous privilege, it seemed like a worthy one.

If we end up needing to do this for the majority of repos, then we should absolutely document it as the default approach, but we're not there yet today.

Reply:
> There's a tradeoff here balancing convenience and "least privilege"

This is exactly my point. Without granular teams per repo and access level, we are reducing our security and going against PLP. Without some sort of automation or config-as-code way to manage the teams, convenience will undermine the goals of the teams mandate.

Reply:
I think that is something we can iterate on as well - automation is worth considering but in practice is often overkill, and the goal here isn't strict PLP, but a safer balance.